collect_app/src/test/java/org/odk/collect/android/utilities/FileUtilsTest.java

```diff
@@ -35,4 +39,79 @@ public void mediaDirNameIsCorrect() {
         assertEquals(expected, FileUtils.constructMediaPath("sample-file.123"));
         assertEquals(expected, FileUtils.constructMediaPath("sample-file.docx"));
     }
+
+    @Test public void identifiesFormMetadata_formSimpleForm() throws IOException {
```

Comment:
Could this be `getMetadataFromFormDefinition_withoutSubmission_returnsMetaDataFields()`?

Reply:
Thanks!